internal/gengapic/client_init.go

```diff
@@ -207,8 +207,8 @@ func (g *generator) clientInit(serv *descriptor.ServiceDescriptorProto, servName
 		p("// Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.")
 		p("type %sClient struct {", servName)
 
-		p("// The connection to the service.")
-		p("conn *grpc.ClientConn")
+		p("// Connection pool of gRPC connections to the service.")
+		p("connp gtransport.ConnPool")
```

Comment:
Totally minor, but I think I prefer `connPool`. Not clear what `connp` stands for. Feel free to ignore.

Reply:
done